e2e/web.test.ts

```diff
@@ -121,4 +121,26 @@ forEachCli(currentCLIName => {
       }
     });
   });
+
+  describe('CLI - Environment Variables', () => {
+    it('should support NX environment variables', () => {
+      ensureProject();
+
+      const appName = uniq('app');
+      const envVars = 'NX_BUILD=52 NX_API=QA';
+      const main = `apps/${appName}/src/main.ts`;
+      const newCode = `const envVars = [process.env.NODE_ENV, process.env.NX_BUILD, process.env.NX_API];`;
+
+      runCLI(`generate @nrwl/web:app ${appName} --no-interactive`);
+
+      const content = readFile(main);
+
+      updateFile(main, `${newCode}\n${content}`);
+
+      runCLI(`build ${appName}`, { envVars });
+      expect(readFile(`dist/apps/${appName}/main.js`)).toContain(
+        'var envVars = ["development", "52", "QA"];'
```

Comment:
Once you get this test running, you'll find that this should be:

```suggestion
        'var envVars = ["test", "52", "QA"];'
```